README.md

```diff
@@ -440,12 +442,13 @@ public function getId(int mobNo) returns table<Student> {
 
     string sqlString = "SELECT * FROM student WHERE mobNo = ?";
     // Retrieve student data by invoking select remote function defined in ballerina sql client
-    var ret = studentDB->select(sqlString, Student, mobNo);
+    var returnValue = studentDB->select(sqlString, Student, mobNo);
 
     table<Student> dataTable = table{};
-    if (ret is error) {
+    if (returnValue is error) {
+        log:printError("Error obtaining student ID from the database", err = returnValue);
```

Comment:
Error in obtaining the student ID from the database to retrieve the marks of the student.